pkg/api/v1alpha1/cluster_test.go

```diff
@@ -1883,6 +1885,131 @@ func TestValidateNetworkingCNIPlugin(t *testing.T) {
 				},
 			},
 		},
+		{
```

Comment:
I think this is the wrong test `TestValidateNetworkingCNIPlugin`
We are not validating the CNI here :)

Reply:
yeah the name is outdated I think